### DISCUSSION TITLE
[feature-request] Toggle Button

### DISCUSSION THREAD

**Discussion by jaaneh** [mantinedev/mantine#3347] [Feature requests]
Hello,

I'd like to see the addition of a toggle button. For my current project this is needed with the ability to select multiple at the same time, and I cannot find anywhere that Mantine supports this
ref: [MUI](https://mui.com/material-ui/react-toggle-button/), [Radix](https://www.radix-ui.com/docs/primitives/components/toggle-group)

**Comment by rtivital** [comment]
It is in the backlog, most likely it will be added in 6.0 release
